tests/foreman/ui/test_subscription.py

```diff
@@ -261,7 +259,7 @@ def test_positive_access_manifest_as_another_admin_user(test_name):
     ).create()
     # use the first admin to upload a manifest
     with Session(test_name, user=user1.login, password=user1_password) as session:
-        manifests.upload_manifest_locked(org.id)
+        target_sat.upload_manifest(org.id)
```

Comment:
Same issue as above, this will need either a manifest fixture or `function_entitlement_manifest_organization`.

Reply:
I made the changes in upload_manifest method itself and it's working. Will that be good?

Reply:
Yes, that's fine for now.